src/db.py

```diff
@@ -86,9 +94,28 @@ def overwrite_student_roster(cid, student_ids):
 	return mongo.db.courses.update({"_id": cid}, {"$set": {"student_ids": student_ids}})
 
 
-def get_assignments_for_course(cid, visible_only=True):
+def get_assignments_for_course(cid, visible_only=False):
 	if visible_only:
-		return list(mongo.db.assignments.find({"course_id": cid, "visibility": True}))
+		now = util.now_timestamp()
+		return list(mongo.db.assignments.find({
```

Comment:
This code looks a bit untidy. Is this the cleanest way we could be doing this?

Reply:
Maybe you could do something like this:
```
startDateVisibilityQuery = {
					"visibility": Visibility.VISIBLE_FROM_START,
					"start": {
						"$lte": now
					}
				}
visibilityQuery = {
					"visibility": Visibility.VISIBLE
				}
return list(mongo.db.assignments.find({"course_id": cid, "$or": [startDateVisibilityQuery, visibilityQuery,...]}))
```

Basically move the dicts into variables so there is less indentation to look at.